app/web/js/app.js

```diff
@@ -1,9 +1,6 @@
 require('jquery');
-require('material-design-lite/dist/material.min.js');
-
+require('./dialog.js');
```

Comment:
Not using Typescript? (same applies to all js assets below)

Reply:
I left JS for the next PR (or an addendum to this one).  Reasons for that:
- i'd like to strip out jQuery (given that it's just a bunch of simple clickhandlers really)
- i wanted to get some opinions on whether or not TS makes sense.  It feels to me that it's not worth the time, upkeep & added learning curve for people joining the project given the very limited amount of JS and the lack of any functions that actually would benefit from this.  There are two places in tooltip.js which could benefit, but those are properly guarded already.  Not sure about the old code, but in general i'd like to get rid of most of it.
- i'd like to get rid of the weird form for the language switcher and make those into plain links.  Not sure why it's a form now, so wanted to get your input on that & verify with the PM whether or not taking the time to do this makes sense.  From a semantic point of view the form makes no sense.  From a styling & code point of view i only see extra complications.  If i get rid of it, then that means most of the old JS is insta-gone.